include/ubiquity_motor/motor_parameters.h

```diff
@@ -175,27 +175,31 @@ struct NodeParams {
     double controller_loop_rate;
     std::string wheel_type;
     std::string wheel_direction;
-
+    std::string drive_type;
     std::string left_joint;
     std::string right_joint;
 
     NodeParams()
         : controller_loop_rate(10.0),
           wheel_type("firmware_default"),
           wheel_direction("firmware_default"),
+          drive_type("2wd"),
```

Comment:
So shouldn't this be set to 4wd here? I mean by default keep it 2wd sure, but some config file needs to change it to 4wd when launching and I'm not certain we do that in the current core/base.launch in avalon_bringup. Do we need to make an addition?

Reply:
This may be useful in Avalon branch with a comment saying in master we need to keep it 2wd.   'Someday' we want to merge avalon back to main and we have to be careful to explain all the oddities.